src/projects/index.ts

```diff
@@ -0,0 +1,37 @@
+import type { Phonic } from "../phonic";
+import type { DataOrError } from "../types";
+import type {
+  AddConversationEvalPromptSuccessResponse,
+  ConversationEvalPromptsSuccessResponse,
+} from "./types";
+
+export class Projects {
+  constructor(private readonly phonic: Phonic) {}
+
+  async listEvalPrompts(
+    projectId: string,
+  ): DataOrError<ConversationEvalPromptsSuccessResponse> {
+    const response =
+      await this.phonic.get<ConversationEvalPromptsSuccessResponse>(
+        `/projects/${projectId}/conversation_eval_prompts`,
+      );
+
+    return response;
+  }
+
+  async addEvalPrompt(
```

Comment:
I think these two methods shouldn't be in projects.
In the SDK, as opposed to the REST API, I suggest we "go straight to the resource" and provide the required params.
Let's also stick with the convention of `list()`, `create()`, `get()`.
I suggest:
```
phonic.conversationEvaluationPrompts.list({ projectId })
phonic.conversationEvaluationPrompts.create({ projectId, name, prompt })
```
Need to be careful with "eval" since it's a reserved word in JavaScript (already bit me a couple of times :))